File: HealthSysHub.Web.Utility/Models/HospitalContentInformation.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HealthSysHub.Web.Utility.Models
{
    public class HospitalContentInformation
    {
        public Guid? HospitalContentId { get; set; }
        public Guid? HospitalId { get; set; }
        public string? Description { get; set; }
        public Guid? CreatedBy { get; set; }
        public DateTimeOffset CreatedOn { get; set; }
        public Guid? ModifiedBy { get; set; }
        public DateTimeOffset? ModifiedOn { get; set; }
        public bool IsActive { get; set; }
    }
}

File: HealthSysHub.Web.Pharmacy.UI/src/app/models/hospital-content-information.ts
export interface HospitalContentInformation {
  hospitalContentId?: string | null;
  hospitalId?: string | null;
  description?: string | null;
  createdBy?: string | null;
  createdOn: Date | string;
  modifiedBy?: string | null;
  modifiedOn?: Date | string | null;
  isActive: boolean;
}
